jsk_panda_robot/panda_eus/euslisp/panda-interface.l

```diff
@@ -0,0 +1,237 @@
+(require :robot-interface "package://pr2eus/robot-interface.l")
+(require :franka "package://panda_eus/models/panda.l")
+
+(ros::roseus-add-msgs "franka_msgs")
+(ros::roseus-add-msgs "franka_gripper")
+
+(defclass panda-robot-interface
+  :super robot-interface
+  :slots (gripper-action gazebop
+          error-recovery-act error
+          gripper-grasp-action gripper-move-action gripper-homing-action gripper-stop-action
+          )
+  )
+(defmethod panda-robot-interface
+  (:init
+   (&rest args &key ((:controller-timeout ct) nil))
+   (prog1
+       (send-super* :init :robot panda-robot
+                    :joint-states-topic "joint_states"
+                    :controller-timeout ct args)
+     ;; check whather the environement is Gazebo
```

Comment:
typo: `whather` -> `whether`

Reply:
remove this
[d19204b](https://github.com/jsk-ros-pkg/jsk_robot/pull/1799/commits/d19204ba8670ad0b08cdff432a81638453e71bbe)